cfg_mgmt/btp_application_certificate.py

```diff
@@ -271,3 +280,10 @@ def delete_config_secret(
             gbaas_client.delete_certificate(info.cn, info.id)
 
     return None
+
+
+def test(cfg_factory: model.ConfigFactory):
```

Comment:
Is it intended to include this hunk?
If so, I would rather prefer to move this to a dedicated (unit)test module and mock external services (cfg-mgmt, gbaas-service).